submissions/kt-std/js-dom/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="css/style.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link rel="stylesheet" media="screen and (min-width:300px) and (max-width: 800px)" href="css/mobile.css">
+	<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Roboto:wght@300;600&display=swap" rel="stylesheet"> 
+	<title>JS DOM</title>
+</head>
+<body class="flex">
+	<header class="header flex center">
+		<nav class="nav flex column full center">
+			<div class="menu__icon-container flex full">
+				<img class="menu__icon" id="menu-btn" src="assets/menu.svg" alt="Menu button">
```

Comment:
Don't forget about the semantic elements in your layout or accessibility attributes.
I'd recommend using the `button` tag here or the `role` attribute on this element

Reply:
Thanks a lot!
I've added button tag to wrap the image